rs/ledger_suite/icrc1/index-ng/tests/incompatible_token_type.rs

```diff
@@ -9,40 +9,35 @@ use ic_state_machine_tests::StateMachine;
 
 mod common;
 
-fn index_wasm_u64() -> Vec<u8> {
-    let index_wasm_path = std::env::var("IC_ICRC1_INDEX_WASM_U64_PATH").expect(
-        "The Ledger wasm path must be set using the env variable IC_ICRC1_INDEX_WASM_U64_PATH",
-    );
-    std::fs::read(&index_wasm_path).unwrap_or_else(|e| {
-        panic!(
-            "failed to load Wasm file from path {} (env var IC_ICRC1_INDEX_WASM_U64_PATH): {}",
-            index_wasm_path, e
-        )
-    })
-}
-
-fn index_wasm_u256() -> Vec<u8> {
-    let index_wasm_path = std::env::var("IC_ICRC1_INDEX_WASM_U256_PATH").expect(
-        "The Ledger wasm path must be set using the env variable IC_ICRC1_INDEX_WASM_U256_PATH",
-    );
+fn index_wasm(env_var: &str) -> Vec<u8> {
+    let index_wasm_path = std::env::var(env_var).expect(&format!(
+        "The Index wasm path must be set using the env variable {}",
+        env_var
+    ));
     std::fs::read(&index_wasm_path).unwrap_or_else(|e| {
         panic!(
-            "failed to load Wasm file from path {} (env var IC_ICRC1_INDEX_WASM_U256_PATH): {}",
-            index_wasm_path, e
+            "failed to load index Wasm file from path {} (env var {}): {}",
+            index_wasm_path, env_var, e
         )
     })
 }
 
 #[test]
 #[should_panic(expected = "assertion `left == right` failed: u256 representation is 32-bytes long")]
 fn should_fail_to_upgrade_index_ng_from_u64_to_u256_wasm() {
-    index_install_and_upgrade_test(index_wasm_u64(), index_wasm_u256());
+    index_install_and_upgrade_test(
+        index_wasm("IC_ICRC1_INDEX_WASM_U64_PATH"),
```

Comment:
Nit (feel free to ignore in this case but keep in mind for the future): Copying string literals in multiple places tend to make the code less error-proof. I'd either create a constant for it or create `index_wasm_u256()` and `index_wasm_u64()` using the new `index_wasm()` function, but this case is simply enough that it's good as it is.

Reply:
That absolutely makes sense; done!